### PR TITLE
Rename h1b to h1.

### PR DESCRIPTION
**Pull request by osenft**
```
----------------------
`make prtest` summary:
----------------------
git rev-parse HEAD
6d286aec3301667e42c1ff397dea7f5155c84831
git status
On branch rename-h1b-to-h1
Your branch is up to date with 'origin/rename-h1b-to-h1'.

nothing to commit, working tree clean
```

### CODE REVIEW

File: golf2/src/main.rs [hunk 1]
```diff
@@ -386,23 +386,23 @@ pub unsafe fn reset_handler() {
...
+    h1::usb::USB0.init(&mut h1::usb::EP0_OUT_DESCRIPTORS,
+                        &mut h1::usb::EP0_OUT_BUFFERS,
+                        &mut h1::usb::EP0_IN_DESCRIPTORS,
+                        &mut h1::usb::EP0_IN_BUFFER,
+                        &mut h1::usb::EP1_OUT_DESCRIPTOR,
+                        &mut h1::usb::EP1_OUT_BUFFER,
+                        &mut h1::usb::EP1_IN_DESCRIPTOR,
+                        &mut h1::usb::EP1_IN_BUFFER,
+                        &mut h1::usb::CONFIGURATION_BUFFER,
+                        h1::usb::PHY::A,
```
Comment (jrvanwhy) [inline golf2/src/main.rs]: The alignment of lines 396-405 is off by 1.

File: golf2/src/main.rs [hunk 2]
```diff
@@ -425,9 +425,9 @@ pub unsafe fn reset_handler() {
     #[allow(unused)]
     let flash_test = static_init!(
         flash_test::FlashTest<
-            h1b::hil::flash::FlashImpl<'static, VirtualMuxAlarm<'static, Timels>>>,
+            h1::hil::flash::FlashImpl<'static, VirtualMuxAlarm<'static, Timels>>>,
         flash_test::FlashTest::<
-            h1b::hil::flash::FlashImpl<'static,
+            h1::hil::flash::FlashImpl<'static,
                                        VirtualMuxAlarm<'static, Timels>>>::new(flash));
```
Comment (jrvanwhy) [inline golf2/src/main.rs]: Alignment off by 1 (lines 430-431)

File: golf2/src/main.rs [hunk 3]
```diff
@@ -478,7 +478,7 @@ impl Platform for Golf {
             kernel::ipc::DRIVER_NUM       => f(Some(&self.ipc)),
             dcrypto::DRIVER_NUM           => f(Some(self.dcrypto)),
             nvcounter_syscall::DRIVER_NUM => f(Some(self.nvcounter)),
-            h1b::usb::driver::DRIVER_NUM  => f(Some(self.u2f_usb)),
+            h1::usb::driver::DRIVER_NUM  => f(Some(self.u2f_usb)),
```
Comment (jrvanwhy) [inline golf2/src/main.rs]: Alignment off by 1.

**Review by jrvanwhy — APPROVED** [review]

File: golf2/src/main.rs (hunk 1 above)
Comment (osenft) [reply golf2/src/main.rs]: Oops, well spotted.

File: golf2/src/main.rs (hunk 2 above)
Comment (osenft) [reply golf2/src/main.rs]: You have really good eyes. I'm impressed!

File: golf2/src/main.rs (hunk 3 above)
Comment (osenft) [reply golf2/src/main.rs]: Well spotted.

**Pull request merged**